tests/events/client/test_resource_schema.py

```diff
@@ -2,9 +2,9 @@
 from typing import Dict, Type
 from uuid import uuid4
 
-import pendulum
 import pytest
 from pydantic import ValidationError
+from src.prefect.types._datetime import now
```

Comment:
same here, maybe this is some auto-import weirdness?

Reply:
Yup, this is auto-import weirdness. I can fix it.